core_backend/app/question_answer/routers.py

```diff
@@ -79,28 +87,38 @@
         },
     },
 )
-async def stt_llm_response(
-    generate_tts: bool = Form(...),
+async def voice_search(
+    generate_tts: bool = Form(False),
```

Comment:
Actually, my radical view is that we should remove this option. We should always return a voice file if the user's input was voice. Otherwise they can use another transcription API (or we can expose ours) and then pass the text to our normal /search endpoint. 

I think voice-to-voice would be the expected behaviour from this endpoint and it's what we need for the demo. 

Perhaps remove this and also response model will always be AudioResponse. If any error occurs, we should return an error 500 with what we managed to get so far

Reply:
makes sense! i will keep it explicitly voice to voice then